css/style.css

```diff
@@ -1,19 +1,167 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+/* HEADER */
+.header {
+    display: flex;
+}
+
+.header-image {
+    height: 50px;
+    width: 50px;
+    padding-left: 160px;
+    padding-top: 15px;
+}
+
+/* NAVIGATION */
+.nav-list {
+    display: flex;
+    list-style: none;
+    margin-left: 900px;
```

Comment:
this large margin is making the <nav> disappear off my screen to the right. If you want the header-image to be left aligned and the <nav> to be right-aligned, can you think of a flex-box property you can use on the `header` class instead of this `margin-left property`?